### ISSUE TITLE
Unused variable in Namefinder?

### ISSUE THREAD

**Issue by Seanny123** [nengo/nengo-gui#615]
The variable `self.base_terms` is never used anywhere in the code, despite being assigned from an argument. May I remove it?

**Comment by tcstewar** [comment]
Yup, that looks to be unused.  I believe the initial reason for having that is that I was going to make namefinder work on an as-needed basis, rather than doing a complete search through the model upfront, but that never happened.

**Comment by Seanny123** [comment]
This appears to be resolved in #806 which refactors parts of the backend. Will close once it merged.
